docs/intelligence_center/integrations.md

```diff
@@ -28,6 +28,26 @@ All the instruction to run it are available at the connector GitHub repository:
 SEKOIA is also providing a [Cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [TheHive](https://thehive-project.org/) ecosystem.
 To setup the analyzer please [follow this guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
 
+In a nutshell:
+
+- Get the SEKOIA.IO API Key
+- Download the Cortex Analyzer from github [here](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter)
+- Make sure the python requirements are installed
```

Comment:
I would remove those bullet points since they are already explained in the installation guide.
Moreover the analyzers must not be downloaded and installed in local but on the cortex instance

```suggestion
```